### PR TITLE
[editor] fix: toolbars rtl issues

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/editor/web/src/RichContentEditor/Toolbars/InlineToolbar/InlineToolbar.jsx [hunk 1]
```diff
@@ -6,9 +6,10 @@ import Measure from 'react-measure';
 import ClickOutside from 'react-click-outside';
 import { debounce } from 'lodash';
 import { DISPLAY_MODE } from 'wix-rich-content-editor-common';
-import Styles from '../../../../statics/styles/inline-toolbar.rtlignore.scss';
+import stylesrtlignore from '../../../../statics/styles/inline-toolbar.rtlignore.scss';
```
Comment (lxgreen) [inline packages/editor/web/src/RichContentEditor/Toolbars/InlineToolbar/InlineToolbar.jsx]:
```suggestion
import stylesRtlIgnore from '../../../../statics/styles/inline-toolbar.rtlignore.scss';
```

File: packages/editor/web/statics/styles/static-toolbar.scss [hunk 2]
```diff
@@ -43,12 +43,14 @@
 
 .staticToolbar_responsiveArrow {
   flex: 0 0 40px;
-  margin-right: -9px;
+  // margin-right: -9px;
```
Comment (lxgreen) [inline packages/editor/web/statics/styles/static-toolbar.scss]: never comment code, always delete it. we have git for this

File: packages/editor/web/statics/styles/static-toolbar.scss [hunk 3]
```diff
@@ -43,12 +43,14 @@
...
+  margin: auto;
   height: 100%;
   background: transparent;
   cursor: pointer;
   border: none;
   border-left: 1px solid $toolbar-border-color;
+  border-right: none;
```
Comment (lxgreen) [inline packages/editor/web/statics/styles/static-toolbar.scss]: redundant; the `border: none` 2 lines above should do the work, right?

File: packages/editor/web/src/RichContentEditor/Toolbars/InlineToolbar/InlineToolbar.jsx (hunk 1 above)
Comment (maimonav) [reply packages/editor/web/src/RichContentEditor/Toolbars/InlineToolbar/InlineToolbar.jsx]: fixed.

File: packages/editor/web/statics/styles/static-toolbar.scss (hunk 2 above)
Comment (maimonav) [reply packages/editor/web/statics/styles/static-toolbar.scss]: fixed.

File: packages/editor/web/statics/styles/static-toolbar.scss (hunk 3 above)
Comment (maimonav) [reply packages/editor/web/statics/styles/static-toolbar.scss]: No, is is essential.
Without it, in rtl mode it will add left border to static toolbar.

**Pull request merged**